L1Trigger/L1TMuonEndCap/src/PtAssignmentEngineDxy.cc

```diff
@@ -21,8 +21,8 @@ void PtAssignmentEngineDxy::configure(int verbose, const std::string pbFileNameD
   pbFileNameDxy_ = pbFileNameDxy;
   std::string pbFilePathDxy_ = "L1Trigger/L1TMuon/data/emtf_luts/" + pbFileNameDxy_;
 
-  inputNameDxy_ = "batch_normalization_1_input";
-  outputNamesDxy_ = {"dense_4/BiasAdd"};
+  inputNameDxy_ = "input1";
```

Comment:
Are changes in this file related to the update of the EMTF displaced NN?

Reply:
Yes. The names of the input and output nodes changed in the new protobuf file. The changes are made to adapt to that.